karapace/schema_reader.py

```diff
@@ -247,26 +249,29 @@ def handle_messages(self) -> None:
                 try:
                     key = ujson.loads(msg.key.decode("utf8"))
                 except ValueError:
-                    self._log.exception("Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
+                    self._log.exception("Invalid JSON in msg.key")
+                    self._log.log(TRACE, "Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
                     continue
 
                 value = None
                 if msg.value:
                     try:
                         value = ujson.loads(msg.value.decode("utf8"))
                     except ValueError:
-                        self._log.exception("Invalid JSON in msg.value: %r, key: %r", msg.value, msg.key)
+                        self._log.exception("Invalid JSON in msg.value")
```

Comment:
The same as above, maybe log the key or parts of it.

Reply:
same answer, private data is removed. debugging data is available in the next line. context is available in the exception message.